tasks/main.yml

```diff
@@ -1,12 +1,19 @@
 ---
-- name: Install packages
+
+- include_vars: vars_RedHat_CentOS.yml
+  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+
+- include_vars: vars_SUSE_openSUSE.yml
+  when: 'ansible_os_family == "Suse" or ansible_os_family == "openSUSE Leap"'
```

Comment:
Same as in the PR for targetcli I would request the move of `vars_*` files from `tasks/` directory to `vars/` directory - https://github.com/OndrejHome/ansible.targetcli/pull/4#issuecomment-603202848

Reply:
Sure, will do.